### DISCUSSION TITLE
Any thoughts for a 1.0 release?

### DISCUSSION THREAD

**Discussion by bstrie** [rusqlite/rusqlite#1097] [Questions]
Here in [this thread](https://www.reddit.com/r/rust/comments/7hkw6c/announcing_diesel_100_beta_1/) we're looking at some of the dependencies of the soon-to-be-1.0 Diesel, and of the DB binding libs it looks like MySQL and Postgres will be seeing a 1.0 release soon. Just curious, is there any sort of wishlist for libsqlite3-sys 1.0?

**Comment by jgallagher** [comment]
I think the only major thing on the libsqlite3-sys wishlist is fleshing out SQLCipher support (#219). We have the ability to link against it, but I haven't looked into it enough to know whether that plays nicely with the way we're using cargo features to require minimum versions of SQLite. I'm inclined to think that could be added in a 1.1-ish release, but don't know for sure.

**Comment by traviscross** [comment]
Because of the nature of the changes, I'd suggest we address #318 and #319 before 1.0 as well.

**Comment by traviscross** [comment]
Prior to 1.0, the fix for #313 should probably also go in.

**Comment by rnewman** [comment]
As an enum case, I'd like to see #325, too.
